machine_learning_hep/fitting/helpers.py

```diff
@@ -61,13 +61,16 @@ def __init__(self, database: dict, ana_type: str, file_data_name: str, file_mc_n
         self.bins1_edges_low = ana_config["sel_an_binmin"]
         self.bins1_edges_up = ana_config["sel_an_binmax"]
         self.n_bins1 = len(self.bins1_edges_low)
-        self.bin2_name = ana_config["var_binning2"]
-        self.bin2_gen_name = ana_config["var_binning2_gen"]
-        self.bins2_edges_low = ana_config["sel_binmin2"]
-        self.bins2_edges_up = ana_config["sel_binmax2"]
-        self.n_bins2 = len(self.bins2_edges_low)
+        self.bin2_name = ana_config.get("var_binning2", None)
```

Comment:
hi @benedikt-voelkel @LuukVermunt. I still get errors if e.g. sec_peak is not defined 

    fit_pars["include_sec_peak"] = self.include_sec_peak[ibin2][ibin1]
TypeError: 'NoneType' object is not subscriptable

Reply:
Happens when `include_sec_peak` is set to `Null` in database. Fixed below

Reply:
thanks @LuukVermunt